README.md

```diff
@@ -1,309 +1,307 @@
-for a while. Please open an issue ticket on this repo if you have problems.
-Cheers.
+Using `zenoh` bridges to pipe the necessary ROS 2 messages between each robot and the `free_fleet_adapter`, users have the flexibility to configure and customize their network setups accordingly following the [official guide](https://github.com/eclipse-zenoh/zenoh?tab=readme-ov-file#configuration-options). Examples provided in this repository are using [these configurations](./free_fleet_examples/zenoh_configs/), do take note of the selective topics that are required for the `free_fleet_adapter` to work. The `zenoh` configuration conveniently allows users to filter and limit the rate of messages based on topics as well, which will be helpful in deployments with limited network bandwidth.
 
-</br>
-</br>
+![](../media/architecture.jpg)
 
-## Installation Instructions
+Each robot's navigation stack is expected to be non-namespaced, while its `zenoh` bridge is expected to be set up with it's robot name as the namespace. This allows the `free_fleet_adapter` to integrate with each robot in the fleet individually using `zenoh` namespaces.
 
-### Prerequisites
+Supports
+* [Ubuntu 24.04](https://ubuntu.com/blog/ubuntu-desktop-24-04-noble-numbat-deep-dive)
+* [ROS 2 Jazzy](https://docs.ros.org/en/jazzy/index.html)
+* [rmw-cyclonedds-cpp](https://github.com/ros2/rmw_cyclonedds)
+* [Open-RMF on main](https://github.com/open-rmf/rmf)
+* [zenoh-bridge-ros2dds v1.0.1](https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds/releases/tag/1.0.1)
+* [zenoh router](https://zenoh.io/docs/getting-started/installation/#ubuntu-or-any-debian)
 
-* [Ubuntu 20.04 LTS](https://releases.ubuntu.com/20.04/)
-* [ROS1 - Noetic](https://wiki.ros.org/noetic)
-* [ROS2 - Galactic](https://docs.ros.org/en/galactic/index.html)
+We recommend setting up `zenoh-bridge-ros2dds` with the released standalone binaries. After downloading the appropriate released version and platform, extract and use the standalone binaries as is. For source builds of `zenoh-bridge-ros2dds`, please follow the [official guides](https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds).
 
-Install all non-ROS prerequisite packages,
+Most of the tests have been performed using `rmw-cyclonedds-cpp`, while other RMW implementations have shown varying results. Support and testing with other RMW implementations will be set up down the road.
 
-```bash
-sudo apt update && sudo apt install \
-  git wget qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools \
-  python3-rosdep \
-  python3-vcstool \
-  python3-colcon-common-extensions \
-  # maven default-jdk   # Uncomment to install dependencies for message generation
-```
+> [!WARNING]
+> This has so far only been tested in simulation, and will undergo updates and changes as more testing is performed. Use at your own risk. For the legacy implementation, check out the [`legacy`](https://github.com/open-rmf/free_fleet/tree/legacy) branch.
```

Comment:
If I'm not wrong you've been testing this on hardware? If so this could be updated 😄